source/postprocess/dynamic_topography.cc

```diff
@@ -41,7 +41,8 @@ namespace aspect
                                quadrature_formula,
                                update_values |
                                update_gradients |
-                               update_q_points);
+                               update_q_points |
+                               update_JxW_values);
```

Comment:
Do you have any idea why this was not necessary previously? It seems to me to be a bug that we didn't require it (though it didn't lead to any wrong results).